docs/getting-started.md

```diff
@@ -120,12 +126,13 @@ Pushing image 172.30.1.1:5000/myproject/nodejs:latest ...
 Push successful
 ```
 
-Great news! Your component has been deployed on OpenShift now. Let's quickly check how it looks!
+Great news! Your component has been deployed to OpenShift! Now we'll connect to the component.
 
-##### Connect to the component
+### 4. Accessing the component
 
-We need create the URL so we can connect to it our application.
-```
+To access the component, we'll need to create an OpenShift endpoint:
```

Comment:
In OpenShift and Kubernetes endpoint means something completely different. We should use `endpoint` in this context.

Reply:
you mean `route` ?